assets/app/scripts/controllers/newfromtemplate.js

```diff
@@ -17,9 +17,9 @@ angular.module('openshiftConsole')
       $location.url(redirect);
     }
 
-    var dcContainers = $parse('template.controllerTemplate.podTemplate.desiredState.manifest.containers');
-    var stiBuilderImage = $parse('parameters.strategy.stiStrategy.image');
-    var outputImage = $parse('parameters.output.to.name || parameters.output.DockerImageReference');
+    var dcContainers = $parse('spec.template.spec.containers');
+    var stiBuilderImage = $parse('spec.strategy.sourceStrategy.from.name');
```

Comment:
we don't just want the names, we need to take the "from" and run it through the `imageObjectRef` filter